api/src/main/java/net/wesjd/anvilgui/AnvilGUI.java

```diff
@@ -461,12 +461,20 @@ public interface ResponseAction extends BiConsumer<AnvilGUI, Player> {
          */
         static ResponseAction replaceInputText(String text) {
             return (anvilgui, player) -> {
-                final ItemStack outputSlotItem =
-                        anvilgui.getInventory().getItem(Slot.OUTPUT).clone();
-                final ItemMeta meta = outputSlotItem.getItemMeta();
+                ItemStack item = anvilgui.getInventory().getItem(Slot.OUTPUT);
+                if (item == null) {
+                    // Fallback on left input slot if player hasn't typed anything yet
+                    item = anvilgui.getInventory().getItem(Slot.INPUT_LEFT);
+                }
+                if (item == null) {
+                    return; // Do nothing if left input slot is also empty
```

Comment:
Do you think that this should instead throw an exception? After all, the developer is stating to change the input text, but there is no input text to change...